docs/admin/observability/alerting.mdx

```diff
@@ -7,31 +7,31 @@ Alerts can be configured to notify site admins when there is something wrong or
 Alerts fall in one of two severity levels:
 
 - <span class="badge badge-critical">critical</span>: something is _definitively_ wrong with Sourcegraph, in a way that is very likely to be noticeable to users. We suggest using a high-visibility notification channel for these alerts.
-  - **Examples:** Database inaccessible, running out of disk space, running out of memory.
+  - **Examples:** Database inaccessible, running out of memory, gitserver volume 90% full.
   - **Suggested action:** Page a site administrator to investigate.
-- <span class="badge badge-warning">warning</span>: something _could_ be wrong with Sourcegraph. We suggest checking in on these periodically, or using a notification channel that will not bother anyone if it is spammed. Over time, as warning alerts become stable and reliable across many Sourcegraph deployments, they will also be promoted to critical alerts in an update by Sourcegraph.
-  - **Examples:** High latency, high search timeouts.
+- <span class="badge badge-warning">warning</span>: something _could_ be wrong with Sourcegraph, or may go wrong soon. We suggest checking in on these periodically, or using a notification channel that will not bother anyone if it is spammed. Over time, as warning alerts become stable and reliable across many Sourcegraph deployments, they will also be promoted to critical alerts in an update by Sourcegraph.
+  - **Examples:** High latency, high search timeouts, gitserver volume 80% full.
   - **Suggested action:** Email a site administrator to investigate and monitor when convenient, and please let us know so that we can improve them.
 
-Refer to the [alert solutions reference](https://docs.sourcegraph.com/admin/observability/alerts) for a complete list of Sourcegraph alerts, as well as possible solutions when these alerts are firing.
+Refer to the [alert solutions reference](https://docs.sourcegraph.com/admin/observability/alerts) for a complete list of Sourcegraph alerts, as well as possible solutions when these alerts are firing$$.
 Learn more about metrics, dashboards, and alert labels in our [metrics guide](/admin/observability/metrics).
 
 ## Setting up alerting
 
-> NOTE: Feature supported in Sourcegraph 3.17+.
-
 Visit your site configuration (e.g. `https://sourcegraph.example.com/site-admin/configuration`) to configure alerts using the [`observability.alerts`](/admin/config/site_config#observability-alerts) field. As always, you can use `Ctrl+Space` at any time to get hints about allowed fields as well as relevant documentation inside the configuration editor.
 
 Once configured, Sourcegraph alerts will automatically be routed to the appropriate notification channels by severity level.
 
+> NOTE: Both critical and warning alerts should be configured; configuring warning does not automatically enable critical.
+
 ### Example notifiers
 
 #### Slack
 
 ```json
 "observability.alerts": [
   {
-    "level": "critical",
+    "level": "warning",
```

Comment:
Why recommend warning only OOTB? If admins only configure one, they should configure critical first, hence the original recommendation here and below

Reply:
My understanding was that this section gives config examples of the many different notifiers we support. One struggle many of our customer admins have, is they configure one alerting level, but not the other, and all 5 examples previously only showed `"level": "critical"` so I left the most urgent notifiers as critical (PagerDuy, OpsGenie), and changed the 3 less urgent (Slack, email, webhook) as warning, to show a realistic example of how a customer instance would be configured.

Reply:
Looking at it again, for Slack, email, and webhooks, we could show examples of config where critical alerts go to one Slack channel, and warning alerts go to a different Slack channel, etc. I think that'd help our customer admins see this more clearly.

Reply:
I personally think we should include examples for both `critical` and `warning` for every alerting channel example to emphasize that these need to be configured separately.

Reply:
Thanks for context! I think the "less urgent" examples should include both (i.e. either an example for `critical`, or an example with _both_ `critical` and `warning`)

Reply:
I agree with both of you. How's this for a compromise?
Slack: Both
PagerDuty: Critical
Opsgenie: Critical
Webhook: Warning
Email: Both

![screencapture-localhost-3000-docs-admin-observability-alerting-2024-01-09-03_37_48](https://github.com/sourcegraph/docs/assets/7050295/123e8ca4-505a-48b3-887e-334f7cb96dd7)

Reply:
I think webhook should include both if we use `warning` for consistency, but happy as-is also :) thanks!

Reply:
Updated webhook to include both.